### PR TITLE
bpo-37826: Document exception chaining in Python tutorial for errors.

### PR DESCRIPTION
**Pull request by maxking**
https://bugs.python.org/issue37826

Automerge-Triggered-By: @maxking

### CODE REVIEW

**Comment by maxking** [comment]
I am going to add a skip-news label since this is a Documentation only change.

File: Doc/tutorial/errors.rst [hunk 1]
```diff
@@ -267,6 +267,53 @@ re-raise the exception::
    NameError: HiThere
 
 
+.. _tut-exception-chaining:
+
+Exception Chaining
+==================
+
+The :keyword:`raise` statement allows an optional :keyword:`from` which allows
```
Comment (warsaw) [inline Doc/tutorial/errors.rst]: Maybe the second "allows" could be "enables"?

File: Doc/tutorial/errors.rst [hunk 2]
```diff
@@ -267,6 +267,53 @@ re-raise the exception::
...
+    <BLANKLINE>
+    The above exception was the direct cause of the following exception:
+    <BLANKLINE>
+    Traceback (most recent call last):
+      File "<stdin>", line 4, in <module>
+    RuntimeError
+
+The expression following the :keyword:`from` must be either an exception or
+``None``. Exception chaining happens automatically when an exception is raised
+inside an exception handler or :keyword:`finally` section. Although, it can be
```
Comment (warsaw) [inline Doc/tutorial/errors.rst]: That last sentence isn't a full sentence.  How about:

> Exception chaining can be disabled by using the ``from None`` idiom:

**Review by warsaw — CHANGES_REQUESTED** [review]

**Comment by maxking** [comment]
I have made the requested changes; please review again.

**Review by warsaw — APPROVED** [review]

**Pull request merged**